integration/js/utils/WebdriverSauceLabs.js

```diff
@@ -72,6 +71,37 @@ const getSafariCapabilities = capabilities => {
   return cap
 };
 
+const getChromeOptions = capabilities => {
+  let chromeOptions = {};
+  if (capabilities.useFakeMedia) {
```

Comment:
Is this capability set somewhere, or as a command line argument? 
Sorry, could not find it in this change hence asking for more understanding.